config/find-eu-exit-guidance-business.yml

```diff
@@ -1,9 +1,4 @@
 ---
-#
-# NOTE: This is a development copy of the find-eu-exit-guidance-business finder config
```

Comment:
I think we should keep the comment for a few weeks, until we no longer need govuk-app-deployment-secrets

Reply:
Oops, that wasn't intentional. Put back 🙂